clients/src/test/java/org/apache/kafka/clients/NetworkClientTest.java

```diff
@@ -613,12 +613,12 @@ private void sendResponse(ResponseHeader respHeader, Struct response) {
         selector.completeReceive(new NetworkReceive(node.idString(), buffer));
     }
 
-    private void sendThrottledProduceResponse(int correlationId, int throttleMs) {
+    private void sendThrottledProduceResponse(int correlationId, int throttleMs, short version) {
```

Comment:
This is to fix `testThrottlingNotEnabledForConnectionToOlderBroker`. The test forces ApiVersionsResponse to version 5, but relied on the fact that nothing really changed between version 5 and version 8 for PRODUCE responses. With flex versions we need to ensure the response matches the ApiVersions response